content/general/concepts/data-types/data-types.md

```diff
@@ -0,0 +1,35 @@
+---
+Title: 'Data Types'
+Description: 'Data is classified into data types that tell the compiler how the data is intended to be used.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Data Types'
+  - 'Types'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/build-web-apps-with-asp-net'
+---
+
+Data is the foundation of all programs. In programming, data is classified into data types to let the compiler know how the data is being used. 

```

Comment:
> Also, because data types vary in name across different programming languages, it would be helpful to include links to the existing data type entries for each language available on Docs.

Ok. I linked all the entries for data types in the CC docs.